### ISSUE TITLE
Pull OnCore Subjects does not support classic projects

### ISSUE THREAD

**Issue by pbchase** [ctsit/redcap_oncore_client#13]
Pull OnCore Subjects does not support classic projects.  It works only with longitudinal projects.

**Comment by ChemiKyle** [comment]
I'm not seeing any failures that aren't being caused by mismatches between OnCore data and REDCap expected values (e.g. REDCap values entered as dropdown or radio buttons silently reject corresponding OnCore data which is delivered as a string), but that also happens to longitudinal projects.

Possible some change made in PR #18 solved this issue.

**Comment by ChemiKyle** [comment]
Have been unable to reproduce, assuming PR #18 did in fact solve.

**Comment by pbchase** [comment]
No really, I can reproduce this bug in your current PRs, @ChemiKyle.  My test project was built from the Basic Demography template project. The Oncore client would fail. I made the smallest change--I enabled _Use longitudinal data collection with defined events_--and the oncore client started importing subject records.  I did not even define an event. 

Curiously, I disabled _Use longitudinal data collection with defined events_ and was able to import records.  So maybe this is a problem with creating the first records that only exists in classic projects. It's a subtle one. I can see how you might not have been able to reproduce it.
